src/app/lib/config/services/afrique.js

```diff
@@ -75,6 +75,7 @@ export const service = {
         audioPlayer: 'Lecteur audio',
         videoPlayer: 'Lecteur vidéo',
       },
+      gist: '	Points clés',
```

Comment:
Just to double check, some of the translations have a space at the beginning, is this intended? 

I've seen it on translations for these services:
Afrique
Amharic 
Somali
Tamil
Turkce
Ukchina

Reply:
Will fix and push, likely a copying error. Thank you

Reply:
@HarveyPeachey should be fixed in latest commit - good spot! Apologies

Reply:
@pvaliani No worries, it's fine. It's happened before when I've added translations, it's a common thing don't worry